themes/janeswalk/js/components/pages/Dashboard/DashboardMenu.jsx

```diff
@@ -0,0 +1,26 @@
+
+const DashboardMenu = ({name, style}) => {
+  //TODO: Are the menu items based on available links for each CO or common?
+  const menuItems = [ { display: `${name} Walks`, link: '/cityWalks'},
```

Comment:
If these are never updated, we could keep them outside of the component function scope.